.github/workflows/release.yml

```diff
@@ -0,0 +1,31 @@
+name: Release
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  build-metabase:
+    uses: ./.github/workflows/build-metabase.yml
+    with:
+      version: v0.44.0
+  build-driver:
+    uses: ./.github/workflows/build-driver.yml
+    needs: build-metabase
+  publish:
+    runs-on: ubuntu-latest
+    needs: [build-metabase, build-driver]
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v2
```

Comment:
you don't even need to pull code here, since all you're doing in uploading the artifact